### PR TITLE
Preconnecting to font files.

### PR DESCRIPTION
**Pull request by sanjaiyan-dev**
Preconnecting font `href` for faster `dns` connections.

_Extremely sorry if I made any errors._

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @khanhnwin

**Review by domesticmouse — APPROVED** [review]

**Comment by sanjaiyan-dev** [comment]
Can you give me a small amount of time please there are some more changes to be done.

Extremely sorry for the inconvenience

**Comment by sanjaiyan-dev** [comment]
> Can you give me a small amount of time please there are some more changes to be done.
> 
> Extremely sorry for the inconvenience

Done ✨

File: src/_layouts/base.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <meta name="theme-color" content="#ffffff" />
     {% unless page.strip_fonts == true %}
      <link rel="preconnect" href="https://fonts.googleapis.com" />
-     <link rel="preconnect" href="https://use.fontawesome.com" />
+     <link rel="preload" href="https://use.fontawesome.com/releases/v5.15.1/js/all.js" as="script" />
```
Comment (domesticmouse) [inline src/_layouts/base.html:15]: PTAL @parlough @khanhnwin 

I'm curious how this preload will be keep in sync with the version of fontawesome that is included elsewhere in this site?

Comment (parlough) [reply src/_layouts/base.html:15]: Perhaps just an inline comment saying the URL needs to be updated here and below if upgrading Font Awesome.

As for the other version of font awesome we include a copy of, we mostly don't need to worry about updating it and I'd like to remove that usage soon.

**Pull request merged**